src/EventGrid/EventGrid/Az.EventGrid.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.2.3'
+ModuleVersion = '2.0.0'
```

Comment:
please revert this back to 1.2.3, version will be upgraded during release by Azure-PowerShell team

Reply:
Sure.. Fixed. thx